prime-router/src/test/kotlin/azure/observability/event/AzureEventUtilsTest.kt

```diff
@@ -18,15 +19,53 @@ class AzureEventUtilsTest {
 
         val expected = listOf(
             ObservationSummary(
-                ConditionSummary(
-                    "840539006",
-                    "Disease caused by severe acute respiratory syndrome coronavirus 2 (disorder)"
+                listOf(
+                    TestSummary(
+                        listOf(
+                            CodeSummary(
```

Comment:
Can we add a case where there are multiple test performed codes in a single observation?

Reply:
Done. Added some other cases, too.